Makefile

```
@@ -1,101 +1,19 @@
-.PHONY: setup dev test test-unit test-integration test-compliance test-all test-watch lint audit agent-check agent-fix build deploy
+PYTHON ?= python3
+PIP ?= $(PYTHON) -m pip
 
-# ============================================================
-# SETUP
-# ============================================================
-setup:
-	pip install -e ".[dev]"
-	pre-commit install
+.PHONY: install install-dev test new-service predeploy
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)